src/herder/HerderTests.cpp

```diff
@@ -132,8 +134,11 @@ TEST_CASE("standalone", "[herder]")
                     feedTx(b);
                 }
 
-                bool hasC =
-                    app->getLedgerManager().getCurrentLedgerVersion() >= 10;
+                bool hasC = false;
+                {
```

Comment:
I see there are some other uses of this pattern:

```
LedgerState ls(app->getLedgerStateRoot());
read something from ls.loadHeader().current()
```

also some of the form:

```
LedgerState ls(app->getLedgerStateRoot());
write something to ls.loadHeader().current()
ls.commit();
```

and multiple usages that do not seem to follow any of those patterns.

I'm wondering if you have considered extracting two first into some functions, as introducing new scope just for that seems cumbersome, for example:

```
auto hasC = readHeader(app).ledgerVersion;
writeHeader(app, [](Heder &h){ h.ledgerVersion = 11; });
``` 

I think first one could be useful, not sure if second one is worth it.

Reply:
I think I'd rather leave it as is for now since it makes it very explicit that a `LedgerState` is being used (this is important for reasoning about deactivation and nesting of ledger states).